console/templates/newaccount-notification-template.txt

```diff
@@ -1,11 +1,16 @@
 Dear admin,
 
-A new user signed up on {publicUrl}/ !
+A new {oauth2} user signed up on {publicUrl}/ !
```

Comment:
extra space to handle when `{oauth2}` is empty

Reply:
`{publicUrl}/` to be replaced with `{instanceName}` btw

Reply:
But {instanceName} is null when testing

Reply:
> extra space to handle when `{oauth2}` is empty

code updated

Reply:
{instanceName} cannot be null, since it is also used at the end of the template ...